ydb/core/security/secure_request.h

```diff
@@ -137,7 +141,9 @@ class TSecureRequestActor : public TBase {
 
 public:
     bool IsTokenRequired() const {
-        return GetEnforceUserTokenRequirement() || (RequireAdminAccess && !GetAdministrationAllowedSIDs().empty());
+        return GetEnforceUserTokenRequirement()
```

Comment:
don't you think it would be better to rewrite it without multiple ORs. just 3 if's with corresponding comments?